custom-vu/src/main/kotlin/jces1209/vu/page/DcIssuePage.kt

```diff
@@ -57,4 +60,53 @@ class DcIssuePage(
     override fun linkIssue(): DcIssueLinking {
         return DcIssueLinking(driver)
     }
+
+    override fun changeAssignee(): DcIssuePage {
+        var assignee = "Tdztbchh Eskfsyas"
+
+        driver
+            .findElement(By.cssSelector("[id = 'assignee-val']"))
+            .click();
+
+        val userPiccker = driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id=\"assignee-form\"]"))
+            )
+
+        Actions(driver)
+            .sendKeys(Keys.DELETE, Keys.ARROW_DOWN)
+            .perform()
+
+        driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id=\"assignee-group-suggested\"]"))
+            )
+
+        var selectedUser = driver
+            .wait(
+                ExpectedConditions.elementToBeClickable(By.xpath("//*[@id=\"all-users\"]/li[3]/a"))
+            )
+
+        var user = selectedUser.text.substringBefore(" - ")
+
+        Actions(driver)
+            .sendKeys(user)
+            .perform()
```

Comment:
What about just to make `.sendKeys(Keys.DELETE, Keys.ARROW_DOWN, Keys.ARROW_UP, Keys.ENTER, Keys.ENTER)`?
and wait for invisibilityOfAllElements By.id("assignee-form")

Reply:
I've tried such approach. Browser should have some time to process event and catch required user up.